guide/payments/request.md

```diff
@@ -116,52 +108,126 @@ A piece of meta data unique to invoices is an expiration time. This is how long
+
+#### Long expiry times
+
+Longer expiry times are useful for when a payment can not be made immediately. Longer times expose users to price volatility which may result in incorrect payment amounts being sent or received. It is recommended that when using a longer expiry amount that the sender defines the amount to limit exposure to price volatility.
+
+A hybrid approach your application could take is when an invoice with a defined amount is generated it uses a short expiry time. If this invoice is shared externally the amount is set to zero and a larger expiration time is used. If a invoice is generated with no amount it automatically uses a longer expiry time.
+
+<div class="image-slide-gallery">
+
 {% include picture.html
-   image = "/assets/images/guide/payments/request/single-use-transaction-request.png"
-   retina = "/assets/images/guide/payments/request/single-use-transaction-request@2x.png"
+   image = "/assets/images/guide/payments/request/single-use-payment-request.png"
+   retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
    layout = "shadow"
-   caption = "A newly generated single-use on-chain payment request."
-   alt-text = "Screen showing newly generated single-use on-chain payment request."
+   caption = "Shorter expiry's are better suited with a defined amount and invoices that refresh each time one expires."
+   alt-text = "Screen showing invoice with short expiry."
    width = 250
    height = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/single-use-transaction-request-settings.png"
-   retina = "/assets/images/guide/payments/request/single-use-transaction-request-settings@2x.png"
+   image = "/assets/images/guide/payments/request/long-expiry.png"
+   retina = "/assets/images/guide/payments/request/long-expiry@2x.png"
    layout = "shadow"
-   caption = "Let users configure the settings of the single-use on-chain payment request."
-   alt-text = "Screen showing newly generated single-use on-chain payment settings."
+   caption = "Longer expiry's are better suited with no defined amount to avoid exposure to price volatility."
+   alt-text = "Screen showing invoice with long expiry."
    width = 250
    height = 541
 %}
 
+</div>
+
+Some other invoice expiry design considerations to consider include; Allowing advanced users to set their own custom expiration times in the settings. Or make invoice expirations a thing of the past and adopt static [Bolt 12](https://bolt12.org/) offers in your application. Offers are not yet a BOLT or widely supported by wallets so interoperability will be an issue adopting it.
+
+### Address types
+
+For on-chain, there are several address types that can be used for single-use payment requests.
+
+<div class="center" markdown="1">
+
 {% include picture.html
    image = "/assets/images/guide/payments/request/single-use-address-type.png"
```

Comment:
Would love to see more useful descriptions here. "Do more with your bitcoin" is pretty meaningless (that's what Muun uses in their Taproot upgrade flow, I think). The logical option for anyone here is to use SegWit, as that has a clear benefit of saving on fees.